frontend/src/components/Dashboard.vue

```diff
@@ -108,13 +155,31 @@ export default {
             console.error(reason);
             return Promise.reject(reason);
           });
+    },
+    toggleVisibility(key) {
+      const indexIfThisKeyIsHidden = this.hiddenElements.indexOf(key);
+      if (indexIfThisKeyIsHidden !== -1) {
+        this.hiddenElements.splice(indexIfThisKeyIsHidden, 1);
+      } else {
+        this.hiddenElements.push(key);
+      }
+      preferences.hiddenJobs = this.hiddenElements
```

Comment:
is it possible to make it generics? **hiddenJobs** is related to workflows

Reply:
Solved. There is now a hiddenElements object, which currently only holds an entry 'jobs' which are all the hidden jobs. If another dashboard would use this feature, it would store its hidden elements in a different key.